script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1391,109 +1586,133 @@ I don't give a damn whether she was being serious or just trying to rile me up.
 -- Page 254, Offset 22789.
 -- 「あのですね。自分が何を言っているのか理解していますか？　社会を守る私たちより、自由気ままに動いては街を混乱に陥れる、あのはた迷惑な真祖を信じるって言ったんですよ？
-"Excuse me? Do you even understand what you're saying? Did you just tell me that you'd believe that irksome True Ancestor who plunged this city into chaos on a whim over people like us who are protecting society?
+"Excuse me?
+ Can you even hear yourself right now?
+ She plunged this city into chaos on a whim.
+ Did you seriously just tell me that you believe that devil-may-care True Ancestor over us,
+ the people who are actually working to %{i}protect%{/i} society?
 }
 [sha:0fe781b7dde00087e043c2a08653540eca0b99b4]{
 -- Page 254, Offset 22790.
 -- さすがにそれは聞き逃せません。っていうか、わたしをバカにしてるんですか!?」
-Hoh, I'm sure I must have misheard. Are you making fun of me?!"
+I'm sure I must have simply misheard you.
+ Or are you actively trying to make fun of me?!"
 }
 [sha:c79dbe32287e9cec99c431b7f934aabd8847686b]{
 -- Page 255, Offset 22791.
 -- 「バカにしてるんじゃなくて、アルクェイドを信じるって言ってるだけでしょう!?
-"I'm not making fun of you, I'm simply telling you that I trust Arcueid!
+"I'm not making fun of you, all I'm saying is that I trust Arcueid!
 }
 [sha:7a6a0429c1c1b29cfa87db51653ce215621c86b3]{
 -- Page 255, Offset 22792.
 -- 先輩こそどうして分かってくれないんですか!?　アイツと協力すれば、街はすぐに元通りになるじゃないですか！」
-Why don't you, of all people, understand me?! If you two worked together, the city would immediately be back to normal!"
+Why don't you, of all people, understand me?!
+ If you two worked together, the city would be back to normal in no time!"
 }
 [sha:046eb5041853658dc403cbc01c9ac4ad58eef29f]{
 -- Page 256, Offset 22793.
 -- 「彼女と協力するのなんて死んでもお断りです。月を重しにして眠った方がまだマシですから」
 // A throwback to file 05_01_CIEL05_3D line 271 where Arcueid says あいつに手を貸すぐらいなら、太陽を抱いて眠ってもいいぐらい (I'd rather breathe my last while embracing the sun than help her)
-"I'd rather die than cooperate with her. I'd rather breathe my last with the moon pressing down on me."
+"I'd rather die than cooperate with her.
+ I'd let the moon crush the life out of me before I stooped to that."
 }
 [sha:e1c0a489833a0e6a5b167f31704a998804c6dbd9]{
 -- Page 256, Offset 22794.
 -- む。なんか、どこかで聞いたような言い回しだった。
-Huh, that's a line I heard somewhere else.
+Huh, that line feels a little familiar for some reason.
 }
 [sha:c58cd3b9c9daa51dd687caadfd8a820abc68ab32]{
 -- Page 257, Offset 22795.
 -- 「ええ、そうですか。死んだ方がマシって話ですか。
-"Oh really, so that's how it is. You'd rather die, huh?
+"Oh, so that's how it is huh? You'd rather die, would you?
 }
 [sha:67c70b4f4b9ee2e39dcb5c5be05391af0564d645]{
 -- Page 257, Offset 22796.
 -- じゃあ俺も言わせてもらいます。アルクェイドを目の敵にする先輩はなんかぜんぜん先輩らしくありません。ヴローヴの方がマシってレベルで、不愉快です」
-Then let me give it to you straight. The Senpai that treats Arcueid like an enemy is nothing like the Senpai I know. That disgusts me even more than Vlov did."
+Then let me give it to you straight.
+ If you're going to treat Arcueid like an enemy, then you're not the upperclassman I thought I knew at all.
+ You're one that disgusts me even more than Vlov did."
```

Comment:
I think you should just use Senpai here since he's referring to her qualities as herself, rather than as an upperclassman.
Third sentence is a little odd. 不愉快, despite how it looks, is actually pretty colloquial.
>Then let me give it to you straight.
> If you're going to treat Arcueid like an enemy, then you're not the Senpai I thought I knew at all.
> You suck! Even Vlov doesn't suck as much as you!